contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -225,7 +199,7 @@ VRAY_ieProcedural::render()
 	ScopedGILLock giLock;
 	try
 	{
-		object ieCore = g_mainModuleNamespace["IECore"];
```

Comment:
Your commit message implies this was returning `None` sometimes (every time?). Maybe we can test for that and return (or throw) instead.